From: "Barak A. Pearlmutter" <barak+git@cs.nuim.ie>
Date: Fri, 8 Nov 2013 11:12:27 +0000
Subject: logo loco

The file logo.svg contains the haskell logo.  The upstream package
expects it to be installed to the directory in which elisp files to be
loaded are located.  This would be strange on debian, since it would
leave a SVG file in /usr/share/emacs/site-lisp/haskell-mode/logo.svg
and then copied (or linked) to each emacsen flavour
site-lisp/haskell-mode/ directory.  Yuck.

Instead we install it in a standard location for scalable icons.
---
 haskell-process.el | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/haskell-process.el b/haskell-process.el
index b4efba2..80eed79 100644
--- a/haskell-process.el
+++ b/haskell-process.el
@@ -46,7 +46,7 @@
   "Greetings for when the Haskell process starts up.")
 
 (defconst haskell-process-logo
-  (expand-file-name "logo.svg" haskell-mode-pkg-base-dir)
+  "/usr/share/icons/locolor/scalable/haskell.svg"
   "Haskell logo for notifications.")
 
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
